x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -889,4 +891,46 @@ export function trainedModelsRoutes(
         }
       )
     );
+
+  /**
+   * @apiGroup TrainedModels
+   *
+   * @api {post} /internal/ml/inference_models/create_inference_endpoint/:taskType/:inferenceId Create Inference Endpoint
+   * @apiName CreateInferenceEndpoint
+   * @apiDescription Create Inference Endpoint
+   */
+  router.versioned
+    .post({
+      path: `${ML_INTERNAL_BASE_PATH}/inference_models/create_inference_endpoint/{taskType}/{inferenceId}`,
```

Comment:
I reckon we should not mix it with the `trained_models` endpoints. Could you please create a new file, e.g. `inference` and place it there? Also we probably can follow the same API shame as the original ES one https://www.elastic.co/guide/en/elasticsearch/reference/current/put-inference-api.html
```
PUT /_inference/<task_type>/<inference_id>
```

Reply:
Done added in https://github.com/elastic/kibana/pull/180330/commits/c6138c66cd8ac2ec7daf3283a8962e6d49c15911